src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -129,7 +152,7 @@ Dictionary
                 type="radio"
                 name="exampleRadios"
                 id="ciel"
-                value="option1"
+                value="/orgs/CIEL/sources/CIEL/"
```

Comment:
What has this got to do with other sources other than CIEL?

Reply:
from my comment above, when a different source is selected, CIEL radio button is deselected, and `this.sourceUrl` changes to the url of the selected source. If CIEL radio button is selected agin, it gets the  the value `value="/orgs/CIEL/sources/CIEL/"` and assigns it to `this.sourceUrl` again.